unresolved import `notify::FsEventWatcher` with default features

When building your crate with it's default features on Windows you get the following error message:
```
error[E0432]: unresolved import `notify::FsEventWatcher`
   --> C:\Users\Lilly\.cargo\registry\src\index.crates.io-6f17d22bba15001f\clap-adapters-0.2.1\src\reloading.rs:5:14
    |
5   | use notify::{FsEventWatcher, RecursiveMode};
    |              ^^^^^^^^^^^^^^ no `FsEventWatcher` in the root
    |
note: found an item that was configured out
   --> C:\Users\Lilly\.cargo\registry\src\index.crates.io-6f17d22bba15001f\notify-6.1.1\src\lib.rs:214:25
    |
214 | pub use crate::fsevent::FsEventWatcher;
    |                         ^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0432`.
error: could not compile `clap-adapters` (lib) due to 1 previous error
```

It seems FsEventWatcher only exists under macos and the recommended way is to use [RecommendedWatcher](https://docs.rs/notify/4.0.18/notify/type.RecommendedWatcher.html) which aliases to the best watcher based on os.

--- a/src/reloading.rs
+++ b/src/reloading.rs
@@ -2,7 +2,7 @@
 
 use std::sync::Arc;
 
-use notify::{FsEventWatcher, RecursiveMode};
+use notify::{RecommendedWatcher, RecursiveMode};
 use tokio::sync::watch;
 use tokio_stream::Stream;
 
@@ -45,7 +45,7 @@ use crate::traits::FromReader;
 #[must_use = "Dropping the `Reloading` will cancel the file watch"]
 pub struct Reloading<T> {
     reload_rx: watch::Receiver<T>,
-    _watcher: Arc<FsEventWatcher>,
+    _watcher: Arc<RecommendedWatcher>,
 }
 
 impl<T: Clone> Reloading<T> {

